frontend/mgramseva/lib/env/app_config.dart

```diff
@@ -38,3 +39,22 @@ Map<String, dynamic> stageConstants = {
 Map<String, dynamic> prodConstants = {
   _baseUrl: "https://api.production.com/",
 };
+
+class FirebaseConfigurations {
+  static const _apiKey = "AIzaSyBWQkRGvXiKu_fLAA5O8SvQzZTWeQTqMZ8";
+  static const _authDomain = "mgramseva-qa.egov.org.in";
```

Comment:
pls remove this when you are merging to master  add this to .git ignore file

Reply:
sure